### DISCUSSION TITLE
Error: NEXT_NO_VERSION: No Next.js version could be detected in your project.

### DISCUSSION THREAD

**Discussion by vadim-vozmitsel** [vercel/community#2030] [Help] [answered]
### Summary

Currently not able to deploy my turbo to vercel and getting the errors mentioned below consistently:

Error: No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies"

NEXT_NO_VERSION: No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies".

I've gone through the suggested troubleshooting steps such as deleting package.json and pnpm-lock.yaml + node_modules and reinstalling everything, listing next only in the dependencies and not in devDependencies. Pushed all fixes to my Git repo, etc but still every deployment fails.

Is there anything else I can try and/or look at to fix this?

**Comment by swarnava** [comment, marked as answer]
Hi Vedim, We have reached out to you via ticket 00130128.

**Reply by vadim-vozmitsel** [reply]
Hello @swarnava, thank you very much for the quick turnaround. I responded via email with the requested info.
